ci-operator/config/smatula/ansible-tests/smatula-ansible-tests-master__dotnet-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .
+  to: dotnet-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: dotnet-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+    test:
+    - ref: dotnet-execute-tests
+    - as: wait
```

Comment:
Adding the wait here may never get used if the dotnet-execute-tests ref fails. If your adding the wait so that you can manually interacted with the cluster I'd add it in the execute-tests ref itself.